pwiz_tools/Skyline/Model/Irt/Regression.cs

```diff
@@ -0,0 +1,197 @@
+﻿/*
+ * Original author: Kaipo Tamura <kaipot .at. uw.edu>,
+ *                  MacCoss Lab, Department of Genome Sciences, UW
+ *
+ * Copyright 2020 University of Washington - Seattle, WA
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using pwiz.Common.DataAnalysis;
+using pwiz.Skyline.Model.DocSettings;
+using pwiz.Skyline.Model.RetentionTimes;
+using pwiz.Skyline.Properties;
+using pwiz.Skyline.Util;
+
+namespace pwiz.Skyline.Model.Irt
+{
+    public static class IrtRegressionType
```

Comment:
You should probably extend "LabeledValues" here so that you get a "Nick enum" where you keep track of the non-localized name and the localized label.
You definitely do not want to be writing localized regression names to the database.